### DISCUSSION TITLE
How to cancel "Held messages" and "Speed limitation per ISP" ?

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#2007] [Help with using Postal]
Hello !

I have two questions please.

1) How not to check the emails sent by API (via Mailwizz) to Postal to speed up the process because I want to send a lot of requests via API (via Mailwizz) and also never put any messages in "Held" status?

2) Is it possible to put a speed limit per domain name, for example for Gmail put 3mails/seconds and Hotmail 6mails/seconds ? (Or something similar)

Thank you in advance for your help 😃

**Comment by willpower232** [comment]
Postal holds messages for various reasons but I don't think it would slow down lots of requests.

It is only possible to apply a limit to the whole server which applies to messages being sent, not per receiving domain.

**Reply by Orminor77** [reply]
Thanks for all answer, but there is no way for desactivate the "Held tabs/status" ?

What you think about this comment (https://github.com/postalserver/postal/issues/246#issuecomment-312782008) this is possible to do with Postal ?

**Reply by Orminor77** [reply]
Any idea? please
